.gitpod.yml

```
@@ -10,29 +10,32 @@ tasks:
     command: |
       gp sync-done build_complete
       DANGEROUSLY_DISABLE_HOST_CHECK=true npm run start
+
   - name: backend
     before: echo PIP_USER=no >> ~/.bashrc && export PIP_USER=no
     init: |
+      pip install -U pip --user
```

Maybe install a specific version of pip to avoid accidental breakage? Not a big deal, but can prevent some headaches:

```suggestion
      pip install "pip==23.0.1" --user
```